unittests/test_errors.py

```diff
@@ -2,163 +2,160 @@
-    self.assertEqual(s.errdev('GET', 'ignored'), 'NULL')
-    self.assertEqual(s.errdev('SET', 'foo.txt'), 'foo.txt')
-    self.assertEqual(s.errdev('bar.txt'), 'bar.txt')
-    self.assertEqual(s.errdev(), 'bar.txt')
-    self.assertEqual(s.errdev('SET', 'SCREEN'), 'SCREEN')
-    self.assertEqual(s.errdev(), 'SCREEN')
+    assert s.errdev() == 'NULL'
+    assert s.errdev('GET') == 'NULL'
+    assert s.errdev('GET', 'ignored') == 'NULL'
+    assert s.errdev('SET', 'foo.txt') == 'foo.txt'
+    assert s.errdev('bar.txt') == 'bar.txt'
+    assert s.errdev() == 'bar.txt'
+    assert s.errdev('SET', 'SCREEN') == 'SCREEN'
+    assert s.errdev() == 'SCREEN'
 
-    self.assertEqual(s.errdev('SET', 'NULL'), 'NULL')
-    self.assertEqual(s.errdev(), 'NULL')
+    assert s.errdev('SET', 'NULL') == 'NULL'
+    assert s.errdev() == 'NULL'
 
     default = 'SHORT, LONG, EXPLAIN, TRACEBACK, DEFAULT'
-    self.assertEqual(s.errprt(), default)
-    self.assertEqual(s.errprt('GET'), default)
-    self.assertEqual(s.errprt('GET', 'ignore'), default)
-    self.assertEqual(s.errprt('SET', 'NONE'), 'NONE')
-    self.assertEqual(s.errprt('get'), '')
-    self.assertEqual(s.errprt('SET', 'SHORT, TRACEBACK'), 'SHORT, TRACEBACK')
-    self.assertEqual(s.errprt(), 'SHORT, TRACEBACK')
-    self.assertEqual(s.errprt('LONG'), 'LONG')
-    self.assertEqual(s.errprt(), 'SHORT, LONG, TRACEBACK')
-    self.assertEqual(s.errprt('SET', 'NONE'), 'NONE')
-    self.assertEqual(s.errprt(default), default)
-    self.assertEqual(s.errprt(), default)
+    assert s.errprt() == default
+    assert s.errprt('GET') == default
+    assert s.errprt('GET', 'ignore') == default
+    assert s.errprt('SET', 'NONE') == 'NONE'
+    assert s.errprt('get') == ''
+    assert s.errprt('SET', 'SHORT, TRACEBACK') == 'SHORT, TRACEBACK'
+    assert s.errprt() == 'SHORT, TRACEBACK'
+    assert s.errprt('LONG') == 'LONG'
+    assert s.errprt() == 'SHORT, LONG, TRACEBACK'
+    assert s.errprt('SET', 'NONE') == 'NONE'
+    assert s.errprt(default) == default
+    assert s.errprt() == default
 
     #### chkin, chkout, trcdep, trcnam, qcktrc, sigerr with Python exceptions
 
     s.erract('set', 'exception')
-    self.assertEqual(s.erract(), 'EXCEPTION')
+    assert s.erract() == 'EXCEPTION'
     s.erract('SET', ' ExcEptIon  ')
-    self.assertEqual(s.erract(), 'EXCEPTION')
+    assert s.erract() == 'EXCEPTION'
     s.errdev('set', 'screen')
-    self.assertEqual(s.errdev(), 'SCREEN')
+    assert s.errdev() == 'SCREEN'
 
     s.chkin('zero')
     s.chkin('one')
     s.chkin('two')
     print(s.qcktrc())
-    self.assertEqual(s.trcdep(), 3)
-    self.assertEqual(s.trcnam(0), 'zero')
-    self.assertEqual(s.trcnam(1), 'one')
-    self.assertEqual(s.trcnam(2), 'two')
-    self.assertEqual(s.qcktrc(), 'zero --> one --> two')
+    assert s.trcdep() == 3
+    assert s.trcnam(0) == 'zero'
+    assert s.trcnam(1) == 'one'
+    assert s.trcnam(2) == 'two'
+    assert s.qcktrc() == 'zero --> one --> two'
 
     print()
```

Comment:
For future changes, I would really like to get rid of prints that go to stdout. It makes reading the test report annoying because there's all this extra text that looks like failures instead of it just reporting a nice success.